From c39f4301a826a2307c36296a72655ff7ce7dc984 Mon Sep 17 00:00:00 2001
From: Alexandru Nemes <alexandru.nemes@enea.com>
Date: Thu, 10 Aug 2017 17:49:21 +0300
Subject: [PATCH] aarch 64 dockerfile

Signed-off-by: Alexandru Nemes <alexandru.nemes@enea.com>
---
 docker/Dockerfile | 22 +++++++++-------------
 1 file changed, 9 insertions(+), 13 deletions(-)

diff --git a/docker/Dockerfile b/docker/Dockerfile
index c4b1121..dbf5281 100644
--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,6 +1,6 @@
-FROM ubuntu:14.04
-MAINTAINER Leo Wang <grakiss.wanglei@huawei.com>
-LABEL version="0.1" description="OPNFV Dovetail Docker Container"
+FROM aarch64/ubuntu:16.04
+MAINTAINER Armband team <armband@enea.com>
+LABEL version="0.1" description="OPNFV Dovetail AArch64 Docker Container"
 
 ARG BRANCH=master
 
@@ -17,19 +17,11 @@ RUN \
         python-dev \
         python-mock \
         python-pip \
-        apt-transport-https \
         wget \
+        sudo \
         --no-install-recommends \
 && \
-    apt-get update
-
-RUN wget -qO- https://get.docker.com/ \
-| \
-    sed 's/-q docker-ce/-q docker-ce=17.03.0~ce-0~ubuntu-trusty/' \
-| \
-    sed 's/edge/stable/' \
-| \
-    sh
+    apt-get update &&  apt-get -y install docker.io
 
 ENV HOME /home/opnfv
 ENV REPOS_DIR ${HOME}/dovetail
@@ -38,6 +30,10 @@ WORKDIR /home/opnfv
 RUN \
     mkdir -p ${REPOS_DIR} \
 && \
+    pip install -U setuptools --upgrade \
+&& \
+    pip install -U wheel \
+&& \
     git config --global http.sslVerify false \
 && \
     pip install git+https://git.opnfv.org/dovetail@$BRANCH#egg=dovetail \
-- 
2.7.4

