From: xudan <xudan16@huawei.com>
Date: Sat, 23 Dec 2017 03:10:19 -0500
Subject: [PATCH] Add AArch64 support for Dovetail docker file

Signed-off-by: xudan <xudan16@huawei.com>
---
 docker/Dockerfile | 32 +++++++++++++++++---------------
 1 file changed, 17 insertions(+), 15 deletions(-)

diff --git a/docker/Dockerfile b/docker/Dockerfile
index c5239db..b4fd9f7 100644
--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,8 +1,16 @@
-FROM ubuntu:14.04
-MAINTAINER Leo Wang <grakiss.wanglei@huawei.com>
-LABEL version="0.1" description="OPNFV Dovetail Docker Container"
+FROM aarch64/ubuntu:16.04
+MAINTAINER Armband team <armband@enea.com>
+LABEL version="0.1" description="OPNFV Dovetail AArch64 Docker Container"
+
+RUN apt-get update

 ARG BRANCH=master
+ARG SUB_SOURCE_FILE=/etc/apt/sources.list.d/dovetail.list
+
+RUN touch $SUB_SOURCE_FILE
+RUN echo "deb [arch=arm64] https://download.docker.com/linux/ubuntu/ xenial test" >> ${SUB_SOURCE_FILE}
+#preinstalling apt-transport-https in order to install docker-ce from the above repo
+RUN apt-get install -y apt-transport-https

 RUN \
     apt-get update \
@@ -17,21 +25,11 @@ RUN \
         python3-dev \
         python3-mock \
         python3-pip \
-        apt-transport-https \
         wget \
+        sudo \
         --no-install-recommends \
 && \
-    apt-get update
-
-RUN easy_install -U setuptools==30.0.0
-
-RUN wget -qO- https://get.docker.com/ \
-| \
-    sed 's/-q docker-ce/-q docker-ce=17.03.0~ce-0~ubuntu-trusty/' \
-| \
-    sed 's/edge/stable/' \
-| \
-    sh
+    apt-get update &&  apt-get -y install docker-ce --allow-unauthenticated

 ENV HOME /home/opnfv
 ENV REPOS_DIR ${HOME}/dovetail
@@ -40,6 +38,10 @@ WORKDIR /home/opnfv
 RUN \
     mkdir -p ${REPOS_DIR} \
 && \
+    pip3 install -U setuptools --upgrade \
+&& \
+    pip3 install -U wheel \
+&& \
     git config --global http.sslVerify false \
 && \
     pip3 install git+https://git.opnfv.org/dovetail@$BRANCH#egg=dovetail \
