From c4a96ad72e11f0259f56e4fcd550a0b5089a3f31 Mon Sep 17 00:00:00 2001
From: Alexandru Nemes <alexandru.nemes@enea.com>
Date: Thu, 20 Apr 2017 15:48:50 +0300
Subject: [PATCH] Dockerfile.aarch64

Signed-off-by: Alexandru Nemes <alexandru.nemes@enea.com>
---
 docker/Dockerfile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docker/Dockerfile b/docker/Dockerfile
index 8cc15e0..bcc756b 100644
--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,6 +1,6 @@
-FROM ubuntu:14.04
-MAINTAINER Leo Wang <grakiss.wanglei@huawei.com>
-LABEL version="0.1" description="OPNFV Dovetail Docker Container"
+FROM: aarch64/ubuntu:16.04
+MAINTAINER Armband team <arband@enea.com>
+LABEL version="0.1" description="OPNFV Dovetail AArch64 Docker Container"
 
 RUN \
     apt-get update \
-- 
2.7.4

